set.mm

```diff
@@ -303875,6 +303878,10 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
   althtmldef "^" as '&uarr;';
   latexdef "^" as "\uparrow"; /*
   latexdef "^" as "\widehat{\ }"; */
+htmldef "^r" as "<IMG SRC='uparrow.gif' WIDTH=7 HEIGHT=19 ALT='^' TITLE='^'>" +
+    "<IMG SRC='r.gif' WIDTH=7 HEIGHT=19 ALT='r' TITLE='r'>" ;
```

Comment:
r.gif is italicized r on the symbols page
http://us2.metamath.org:88/symbols/symbols.html
and has width 8 (you can see the width by hovering over the chart).  If 
you intend this to be a subscript, though, you can use subr.gif which 
has width 5.  In either case I will add it to the build script to copy 
it from the symbols directory to the mpegif directory (I only put the 
needed symbols in mpegif directory).

Mario has done more with compound symbols like this, so he may want to 
comment.

Also, what I sometimes do when a symbol is missing is just use the 
althtmldef and possibly make a symbol for it later.  The primary purpose 
of the mpegif directory was to provide a guaranteed correct display back 
when Unicode was full of browser-dependent bugs and missing characters; 
that is hopefully becoming less of a problem over time.


On 11/29/15 2:57 PM, Jens-Wolfhard Schicke-Uffmann wrote:
> In set.mm <https://github.com/metamath/set.mm/pull/23#discussion_r46097087>:
>
>> @@ -303875,6 +303878,10 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
>>    althtmldef "^" as '&uarr;';
>>    latexdef "^" as "\uparrow"; /*
>>    latexdef "^" as "\widehat{\ }"; */
>> +htmldef "^r" as "<IMG SRC='uparrow.gif' WIDTH=7 HEIGHT=19 ALT='^' TITLE='^'>" +
>> +    "<IMG SRC='r.gif' WIDTH=7 HEIGHT=19 ALT='r' TITLE='r'>" ;
>
> I don't know what r.gif is, or whether it exists (or how to make it). So
> the width is probably wrong.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/metamath/set.mm/pull/23/files#r46097087>.
>